kaitai_utils.py

```diff
@@ -101,28 +128,43 @@ def _object_to_dict(self, instance: Any, path: str) -> Generator[Dict[str, Any],
             elif _is_list(value_object):
                 yield _to_lower_camel_case(key), self._list_to_dict(value_object, path)
             elif isinstance(value_object, bytes):
-                yield self.process_byte_array(value_object, path, key)
+                yield self._process_bytes(value_object, offsets, path, key)
```

Comment:
```suggestion
                yield self._process_bytes(value_object, offsets or {}, path, key)
```

to avoid `offsets` being `None`
(or guard in another way against it?!)

Reply:
good idea, done